desktop/core/src/desktop/js/utils/hooks/useLoadData.ts

```diff
@@ -61,8 +66,16 @@ const useLoadData = <T, U = unknown>(url?: string, options?: IOptions<T, U>): IU
         const fetchUrl = localOptions?.urlPrefix ? `${localOptions.urlPrefix}${url}` : url;
         const response = await get<T, U>(fetchUrl, localOptions?.params, fetchOptions);
         setData(response);
+        setError(undefined);
```

Comment:
setError not required as it is set above.

Reply:
this is required in the edge case.

1. First call made which resulted in error. Hence error is set to non-null and data is unchanged.
2. Second call is made either by triggering `refetch` function or by changing and of the `options`. When its a success, the old error state is still set to non-null and data after second call is also set to non-null.

Reply:
Apologies, I over-sighted the setError at line 63.
Thanks for flagging. I have removed it